keyboards/keychron/k3/config_led.h

```diff
@@ -24,9 +24,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LED_MATRIX_ROWS 6
 #define LED_MATRIX_ROW_CHANNELS 3
 #define LED_MATRIX_ROWS_HW (LED_MATRIX_ROWS * LED_MATRIX_ROW_CHANNELS)
-#define LED_MATRIX_ROW_PINS { C0, C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, C11, C12, C13, C14, D3, B15, B14 }
+#define LED_MATRIX_ROW_PINS { C3, C1, C0, C6, C5, C4, C9, C8, C7, C12, C11, C10, B13, C14, C13, B14, B15, D3 }
```

Comment:
This is way too weird for a keychron config. Are you sure it's not 
`#define LED_MATRIX_COL_PINS { A8, A9, A10, A11, A12, A13, A14, A15, B0, B1, B2, B3, B4, B5, B6, B7 }` ?

Reply:
Yeah. I agree it looks weird. But with this configuration all leds look and work correctly. It was actually taken from K2, which have identical layout. Apparently that's how it is set up within K2 and K3. 
Old config was doing some weird stuff and leds were all over the place, missplaced, lighting up some "ghost leds" and even not lighting up some of them.